using-timescaledb/data-retention.md

```diff
@@ -1,15 +1,21 @@
 # Data Retention [](data-retention)
 
-TimescaleDB allows efficient deletion of old data at the chunk level
-using the `drop_chunks` function.
+As your tables grow, it can become cumbersome to keep all that data
+available in the hypertable, so you will need to remove data that are
+no longer relevant. Since data in hypertables are chunked, a normal
+`DELETE` will be very inefficient and TimescaleDB instead provide the
+`drop_chunks` function that is able to quickly drop chunks that are
+not needed any more.
```

Comment:
Not sure "cumbersome" is the appropriate/accurate term here. Also, DELETEs aren't inefficient because hypertables are chunked; it's simply inefficient the way it works in PostgreSQL (chunking actually helps here even for DELETEs).  So this is not an accurate description.

I suggest something like:

```suggestion
The relevance of time-series data often diminishes over time and therefore it is often desirable to delete old data to save disk space. However, deleting large swaths of data from tables can be costly and slow if done row-by-row using the standard `DELETE` command. The recommended way to delete old data on a hypertable is therefore to use `drop_chunks`, which quickly drops data at the granularity of chunks without incurring the same overhead.
```